client/src/components/RelatedItems/RelatedProducts.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import ProductCard from './ProductCard.jsx';
+import ComparisonModal from './ComparisonModal.jsx';
+
+function RelatedProducts({currentProductInfo, relatedProductsInfo}) {
```

Comment:
I think you're right that it's too early to start adding tests. One test-like feature built into React that we could consider though is [Typechecking with proptypes](https://reactjs.org/docs/typechecking-with-proptypes.html). This feature is pretty easy to learn and can be helpful for identifying when a required prop is missing or when the prop type is incorrect (i.e. string instead of integer)

Reply:
That seems really good Aaron. 
I'll add it in my next PR.